proto/gz/msgs/air_speed_sensor.proto

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (C) 2019 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+*/
+
+syntax = "proto3";
+package gz.msgs;
+option java_package = "com.gz.msgs";
+option java_outer_classname = "AirSpeedSensorProtos";
+
+/// \ingroup gz.msgs
+/// \interface AirSpeedSensor
+/// \brief Definition of an air speed sensor
+
+import "gz/msgs/header.proto";
+import "gz/msgs/sensor_noise.proto";
+
+/// \brief Message that describes an air speed sensor.
+message AirSpeedSensor
+{
+  /// \brief Optional header data
+  Header header               = 1;
+
+  /// \brief Sensor speed noise.
+  SensorNoise speed_noise  = 3;
```

Comment:
If we decide that the SDF PR is needed (https://github.com/gazebosim/sdformat/pull/1215#discussion_r1095907097), we should unify the terminology to make things clear. Either `speed` or `pressure`